SimplenoteUITests/AllNotes.swift

```diff
@@ -7,24 +7,31 @@ class AllNotes {
     }
 
     class func isOpen() -> Bool {
+		//Warning for future: this works only for 'All Notes' obviously
+		//Will not work when a tag is selected.
         return app.navigationBars[UID.NavBar.allNotes].exists
     }
 
     class func open() {
         guard !isOpen() else { return }
 
-        app.navigationBars.element.buttons[UID.Button.menu].tap()
+		Sidebar.open()
         app.tables.staticTexts[UID.Button.allNotes].tap()
     }
 
     class func addNoteTap() {
-        app.navigationBars[UID.NavBar.allNotes].buttons[UID.Button.newNote].tap()
+		app.navigationBars.buttons[UID.Button.newNote].tap()
     }
 
-    class func createNoteAndLeaveEditor(noteName: String) {
-        print(">>> Creating note: " + noteName)
+	class func createNoteAndLeaveEditor(noteName: String, tagsOptional: [String] = []) {
```

Comment:
I'm assuming the name `tagsOptional` means that the tags may not be there?

If that's the case, then you can use the simpler `tags`. Having the default value, `= []` already tells the reader and the Xcode autocompletion that it's not necessary to provide a value for this parameter unless you want to override the default.

```suggestion
	class func createNoteAndLeaveEditor(noteName: String, tags: [String] = []) {
```

Reply:
Fixed, thank you!